tests/helpers/test_condition.py

```diff
@@ -66,3 +67,12 @@ def test_or_condition(self):
 
         self.hass.states.set('sensor.temperature', 100)
         assert test(self.hass)
+
+    def test_time_window(self):
```

Comment:
The test should probably patch `dt_util.now()` like [this](https://github.com/home-assistant/home-assistant/blob/dev/tests/components/automation/test_time.py#L259-L260) so we know what we are testing.